source/threads.tex

```diff
@@ -3146,8 +3144,8 @@
 \begin{itemdescr}
 \pnum
 \expects
-\tcode{order} is
-neither \tcode{memory_order::release} nor \tcode{memory_order::acq_rel}.
+\tcode{order} is either \tcode{memory_order::relaxed}, \tcode{memory_order::consume},
```

Comment:
Can we harmonize this to read "The order argument is ..." (see next change)

Or maybe the other way round.

Reply:
I harmonized it now as of your first suggestion.